src/program/loadtest/find-limit/README

```diff
@@ -1,4 +1,5 @@
 Usage: find-limit [OPTIONS] <PCAP-FILE> <TX-NAME> <RX-NAME> <PCI> [<PCAP-FILE> <TX-NAME> <RX-NAME> <PCI>]...
+       find-limit [OPTIONS] <PCI> <PCAP-FILE>
```

Comment:
One final nit!  Can we reverse these arguments?  I think it's too confusing to have PCI and PCAP be one way in the 4-argument form and the other in the 2-argument form.  We can use PCAP PCI in both.